src/index.html

```diff
@@ -4,9 +4,396 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>  
+      <div class="menu">
+        <a class="logo-link menu__logo-link" href="#">
+          <img 
+            class="logo-link__logo-img" 
+            src="images/logo.png" 
+            alt="logo"
+          >
+        </a>
+        
+        <nav class="navigation menu__navigation">
+          <ul class="navigation__nav-list">
+            <li class="navigation__items">
+              <a class="navigation__link is-active" href="#">
+                Apple
+              </a>
+            </li>
```

Comment:
I'd suggest keeping it simple
```suggestion
        <nav class="nav header__nav">
          <ul class="nav__list">
            <li class="nav__item">
              <a class="nav__link is-active" href="#">
                Apple
              </a>
            </li>
```